### ISSUE TITLE
Command.getRequirements() has a bad return type

### ISSUE THREAD

**Issue by TheTripleV** [robotpy/robotpy-commands-v1#10]
```
>>> wpilib.command.Command().getRequirements()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: Unable to convert function return value to a Python type! The signature was
        (self: wpilib.command._commands_v1.Command) -> wpi::SmallPtrSetImpl<frc::Subsystem * __ptr64>
```
Link to header: https://github.com/wpilibsuite/allwpilib/blob/7275ab98376a01db1ff75aced7260dc6f6f1237e/wpilibOldCommands/src/main/native/include/frc/commands/Command.h#L227

**Comment by auscompgeek** [comment]
Yeah, this is going to need a type caster in robotpy-wpiutil I think. I'm sure I've seen a couple of other instances of SmallSets as well.

**Comment by virtuald** [comment]
This is more than just a type caster I think, since it's a list of pointers.

**Comment by virtuald** [comment]
Fixed in 06b1b0caaaf262190edb3a7d94e7f4a1a3c229d5
